relax/methods/clue.py

```diff
@@ -348,7 +348,7 @@ class CLUE(BaseCFModule, BaseParametricCFModule):
     name: str = 'CLUE'
 
     def __init__(self, m_config: Dict | CLUEConfigs = None):
-        if m_config is None: m_config = CLUEConfigs()
+        if m_config is None or not bool(m_config): m_config = CLUEConfigs()
```

Comment:
You should write:

```python
if m_config is None:
        m_config = CLUEConfigs()
self.m_config = validate_configs(m_config, CLUEConfigs)
```

Reply:
validate_configs doesn't capture the empty dict case, and raises error:
  File "/Users/chuck/PycharmProjects/temp/ReLax/relax/methods/vaecf.py", line 266, in __init__
    self.module = VAECFModule(m_config.dict())
AttributeError: 'dict' object has no attribute 'dict'